factcast-schema-registry-cli/src/main/resources/site/hugo-theme-docdock-master/exampleSite/content/shortcode/icon.md

```diff
@@ -0,0 +1,1259 @@
++++
+draft = false
+title = "icon"
+description = "Display an icon."
+
+[menu.main]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-shortcut-reference-link] Use the trailing [] on reference links](https://app.codacy.com/manual/uwe/factcast/pullRequest?prid=4955738)